fullnode.yaml

```diff
@@ -0,0 +1,43 @@
+base:
```

Comment:
Did u mean to add the configs + genesis blobs?

Reply:
good catch!